cypress/support/commands.ts

```diff
@@ -2,3 +2,25 @@
 Cypress.Commands.add('getByTestId', (testId) => {
   return cy.get(`[data-testid="${testId}"]`);
 });
+
+Cypress.Commands.add('connectWallet', () => {
+  cy.getByTestId('connect-button').as('btn').click();
+  cy.getByTestId('connect-button').contains('Connect Wallet').should('not.exist');
+  cy.get('body').type('{esc}');
+});
+
+Cypress.Commands.add('fillInput', (testId, value) => {
+  cy.get(testId).type(value);
+  cy.get(testId).find('input').should('have.value', value);
+});
+
+Cypress.Commands.add('changeSelect', (selectId, itemId, value) => {
+  cy.get(selectId).click();
+  cy.getByTestId(itemId).click();
+  cy.get(selectId).find('input').should('have.value', value);
+});
+
+Cypress.Commands.add('clickButton', (buttonId) => {
+  cy.get(buttonId).should('be.enabled');
```

Comment:
I think this custom command is not necessary, the logic is very simple and short to abstract it in a custom command. Also, there's no need to check if the button is enable before clicking it, the command will fail anyways if the click could not be performed ([checks here](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress#:~:text=click()%20command%2C%20Cypress%20ensures,Not%20being%20covered)), and the availability of the button is not the main objective of the custom command.

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/6/commits/f9882f145c7df37628625e01dec0391ad6dd9e81